### PR TITLE
boostnote: Update to version 0.23.1, add boostnote-local 0.22.3

### PR DESCRIPTION
**Pull request by chenxiangfang**

### CODE REVIEW

**Comment by rashil2000** [comment]
/verify

Do the binaries `Boost Note.exe` and `Boost Note - Local.exe` take any commandline arguments?

File: bucket/boostnote-local.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        [
+            "Boost Note - Local.exe",
+            "Boost Note - Local"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/BoostIO/BoostNote.next-local"
+    },
+    "autoupdate": {
+        "url": "https://github.com/BoostIO/BoostNote.next-local/releases/download/vv$version/boost-note-local-win.exe#/dl.7z"
```
Comment (rashil2000) [inline bucket/boostnote-local.json:37]:
```suggestion
        "url": "https://github.com/BoostIO/BoostNote.next-local/releases/download/v$version/boost-note-local-win.exe#/dl.7z"
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Review by issaclin32 — DISMISSED** [review]

**Comment by rashil2000** [comment]
There's already a manifest for boostnote.json. Please update that instead of creating new boostnote-app.json

File: bucket/boostnote.json [hunk 2]
```diff
@@ -1,33 +1,38 @@
 {
-    "version": "0.16.1",
-    "description": "A markdown editor for developers",
-    "homepage": "https://boostnote.io",
+    "version": "0.23.1",
+    "description": "A collaborative real-time markdown note app for developer teams",
+    "homepage": "https://github.com/BoostIO/BoostNote-App",
```
Comment (rashil2000) [inline bucket/boostnote.json:4]: We can keep using boostnote.io as we already provide GitHub URL in checkver

File: bucket/boostnote-local.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "0.22.3",
+    "description": "A lightspeed workspace for developers",
+    "homepage": "https://github.com/BoostIO/BoostNote.next-local",
```
Comment (rashil2000) [inline bucket/boostnote-local.json:4]: Same as above

**Pull request merged**